src/modules/job-manager/Makefile.am

```diff
@@ -129,11 +136,8 @@ TESTS = \
 	test_annotate.t
 
 test_ldadd = \
-	libjob-manager.la \
+	${COMMON_LIBS}
 	$(top_builddir)/src/common/libtap/libtap.la \
```

Comment:
Test link to libtap is here.

Reply:
Oh no, I see what you mean, not sure why that worked, I'll add the backslash and amend this.  Thanks!